### DISCUSSION TITLE
Project Doubts

### DISCUSSION THREAD

**Discussion by celions** [TheAlbertDev/MASB#57] [Q&A] [answered]
When talking about the requirements, which is the purpose of assigning a code to them?

Thanks!

**Comment by Albert-Alvarez** [comment, marked as answer]
To have an unequivocal reference to each one of them. This, when using the code, allows to show which requirement is being talked about. It is also useful when it is necessary to make control/monitoring tables where putting the whole specification as a column header, for example, is not feasible. This is how you will find it tomorrow in project specifications.
